airflow/models/taskinstance.py

```diff
@@ -963,12 +963,31 @@ def signal_handler(signum, frame):
                 Stats.incr('ti_successes')
             self.refresh_from_db(lock_for_update=True)
             self.state = State.SUCCESS
+            self.log.info(
+                'Marking task as SUCCESS.'
+                'dag_id=%s, task_id=%s, execution_date=%s, start_date=%s, end_date=%s',
+                self.dag_id,
+                self.task_id,
+                self.execution_date.strftime('%Y%m%dT%H%M%S'),
+                self.start_date.strftime('%Y%m%dT%H%M%S'),
+                self.end_date.strftime('%Y%m%dT%H%M%S'))
         except AirflowSkipException as e:
             # log only if exception has any arguments to prevent log flooding
             if e.args:
                 self.log.info(e)
             self.refresh_from_db(lock_for_update=True)
             self.state = State.SKIPPED
+            try:
+                self.log.info(
+                    'Marking task as SKIPPED.'
+                    'dag_id=%s, task_id=%s, execution_date=%s, start_date=%s, end_date=%s',
+                    self.dag_id,
+                    self.task_id,
+                    self.execution_date.strftime('%Y%m%dT%H%M%S'),
+                    self.start_date.strftime('%Y%m%dT%H%M%S'),
+                    self.end_date.strftime('%Y%m%dT%H%M%S'))
+            except NameError:
```

Comment:
Is the `try-except` really necessary?

Reply:
wasn't sure where skip exception could be thrown so just a railcard

Reply:
No I mean why do you check for `NameError` ?

Reply:
not sure if start_date existed yet